IP Groups in override-rules (#536)
nginx/documentation: content/includes/nap-waf/config/common/ip-groups-overview.md

> IP groups is a feature to organize lists of allowed and forbidden IP addresses in several lists with common attributes.
> With this enhancement, users have more control over how a unique policy setting is applied to incoming requests with a specific IP address.
> Each IP Group contains a unique name, enforcement type (always, never and policy-default), list of IP addresses.

Reviewer: ```suggestion
IP groups is a feature to organize lists of allowed and forbidden IP addresses across several lists with common attributes.

This allows you to control unique policy settings for incoming requests based on specific IP addresses.

Each IP Group contains a unique name, enforcement type (_always_, _never_ and _policy-default_), and list of IP addresses.

```

IP groups is a feature to organize lists of allowed and forbidden IP addresses across several lists with common attributes.

This allows you to control unique policy settings for incoming requests based on specific IP addresses.

Each IP Group contains a unique name, enforcement type (_always_, _never_ and _policy-default_), and list of IP addresses.

An example of a declarative policy using IP Groups configuration: 

```json
{ 
  "policy": { 
    "name": "IpGroups_policy", 
    "template": { 
       "name": "POLICY_TEMPLATE_NGINX_BASE" 
    }, 
    "applicationLanguage": "utf-8", 
    "caseInsensitive": false, 
    "enforcementMode": "blocking", 
    "ip-address-lists": [ 
       { 
         "name": "Standalone",
         "description": "Optional Description",
         "blockRequests": "policy-default",
         "setGeolocation": "IN",
         "ipAddresses": [
          {
             "ipAddress": "1.2.3.4/32"
          },
          {
             "ipAddress": "1111:fc00:0:112::2"
          }
        ]
      }
    ]
  }
}

```

> The example with IP-Group definition in external file external_ip_groups.json 

Reviewer 2: ```suggestion
The example with IP-Group definition in external file external_ip_groups.json:
```
